tests/integration.rs

```diff
@@ -130,3 +130,274 @@ async fn test_app_mention_integration() {
     let context = runtime.db.get_channel_context(channel_id).await.expect("Failed to get context");
     assert!(!context.is_empty() || context.is_empty()); // Check that context exists (can be empty)
 }
+
+#[tokio::test]
+async fn test_context_update_integration() {
+    // Set up the test environment
+    let runtime = setup_test_environment().await;
+
+    // Skip if no API key
+    if runtime.config.inner.openai_api_key == "test_key" {
+        return;
+    }
+
+    let channel_id = "C02CONTEXTTEST";
+    let thread_ts = "1234567890.456789";
+
+    // Create a message that asks for context update
+    let context_update_message = serde_json::json!({
+        "type": "app_mention",
+        "user": "U54321",
+        "text": "<@U12345> Please update the context for this channel with new information about deployment procedures",
+        "ts": "1234567890.456789",
+        "channel": channel_id,
+        "event_ts": "1234567890.456789",
+    });
+
+    // Call the handler
+    triage_bot::interaction::chat_event::handle_chat_event(
+        context_update_message,
+        channel_id.to_string(),
+        thread_ts.to_string(),
+        runtime.db.clone(),
+        runtime.llm.clone(),
+        runtime.chat.clone(),
+    );
+
+    // Wait for processing
+    tokio::time::sleep(tokio::time::Duration::from_secs(5)).await;
```

Comment:
Can you _try_ to fix this so that we don't have to wait?  Live queries are likely a good way to do this.

Establish the query,

```
let mut stream = db.select("person").live().await?;
```

make the call, and then 

```
let Some(result) = stream.next().await
```

And do the same for the others?